consensus/polybft/validator_metadata.go

```diff
@@ -116,13 +117,24 @@ func (v *ValidatorMetadata) UnmarshalRLPWith(val *fastrlp.Value) error {
 
 // fmt.Stringer implementation
 func (v *ValidatorMetadata) String() string {
-	return fmt.Sprintf("Address=%v; BLS Key=%v; Voting Power=%d",
-		v.Address.String(), hex.EncodeToString(v.BlsKey.Marshal()), v.VotingPower)
+	return fmt.Sprintf("Address=%v; Voting Power=%d; BLS Key=%v;",
+		v.Address.String(), v.VotingPower, hex.EncodeToString(v.BlsKey.Marshal()))
 }
 
 // AccountSet is a type alias for slice of ValidatorMetadata instances
 type AccountSet []*ValidatorMetadata
 
+// fmt.Stringer implementation
+func (as AccountSet) String() string {
+	var buf bytes.Buffer
+	for _, v := range as {
+		buf.WriteString(v.String())
+		buf.WriteString("\n")
+	}
+
+	return buf.String()
```

Comment:
Could we just use `strings.Join(as, "\n")` ? It also uses strings.Builder with buffer inside.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1103/commits/126ff2ec8265d399c3b9637f8dab164a488a9b38